### PR TITLE
Feature/sonar part two

### PR DESCRIPTION
**Pull request by Gwen-Bradbury**

### CODE REVIEW

File: tests/SonarPartTwoTest.php [hunk 1]
```diff
@@ -7,8 +7,18 @@
...
 {
-    public function  testGivenOneNumberReturnsZero(): void
+    public function  testGivenOneWindowReturnsSix(): void
     {
         $this->assertSame(6, (new SonarPartTwo())->SweepPartTwo("1\n2\n3"));
     }
+
+    public function  testGivenSecondWindowReturnsNine(): void
+    {
+        $this->assertSame(9, (new SonarPartTwo())->SweepPartTwo("1\n2\n3\n4"));
```
Comment (OYates) [inline tests/SonarPartTwoTest.php]: are these really the values you are wanting?

File: src/SonarPartTwo.php [hunk 2]
```diff
@@ -19,4 +29,15 @@ function SweepPartTwo(string $puzzleInput): int
 //    }
 //    $previousMeasurement = $measurement;
 //}
-//return $counter;
\ No newline at end of file
+//return $counter;
+
+
+//Three passing tests
```
Comment (OYates) [inline src/SonarPartTwo.php]: stop commenting out code

File: src/SonarPartTwo.php [hunk 3]
```diff
@@ -6,10 +6,20 @@ class SonarPartTwo
 {
     function SweepPartTwo(string $puzzleInput): int
```
Comment (OYates) [inline src/SonarPartTwo.php:7]: partTwo? why not just change the existing class?

File: src/SonarPartTwo.php [hunk 4]
```diff
@@ -6,10 +6,20 @@ class SonarPartTwo
 {
     function SweepPartTwo(string $puzzleInput): int
     {
-        return 6;
+        $puzzleInput = explode("\n", $puzzleInput);
```
Comment (OYates) [inline src/SonarPartTwo.php:9]: puzzleInput?

File: .phpunit.result.cache [hunk 5]
```diff
@@ -1 +1 @@
...
...IsBiggerReturnsOne":0,"Advent\\Tests\\DayOneTest::testGivenThreeNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\DayOneTest::testGivenFourNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\DayOneTest::testGivenFiveNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\DayOneTest::testExampleInputReturnsSeven":0,"Advent\\Tests\\DayOneTest::testPuzzleInputReturnsCorrectAnswer":0,"Advent\\Tests\\SonarTest::testGivenOneNumberReturnsZero":0.001,"Advent\\Tests\\SonarTest::testGivenTwoNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\SonarTest::testGivenThreeNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\SonarTest::testGivenFourNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\SonarTest::testGivenFiveNumbersWhereTheSecondIsBiggerReturnsOne":0,"Advent\\Tests\\SonarTest::testExampleInputReturnsSeven":0,"Advent\\Tests\\SonarTest::testPuzzleInputReturnsCorrectAnswer":0,"Advent\\Tests\\SonarPartTwoTest::testGivenOneNumberReturnsZero":0.002,"Advent\\Tests\\SonarPartTwoTest::testGivenOneWindowReturnsSix":0.002,"Advent\\Tests\\SonarPartTwoTest::testGivenSecondWindowReturnsNine":0,"Advent\\Tests\\SonarPartTwoTest::testGivenThirdWindowReturns":0}}
```
Comment (OYates) [inline .phpunit.result.cache:1]: can probably hit ignore this file

Comment (Gwen-Bradbury) [reply .phpunit.result.cache:1]: Added it to .gitignore

File: src/SonarPartTwo.php (hunk 4 above)
Comment (Gwen-Bradbury) [reply src/SonarPartTwo.php:9]: puzzleInput is the input data from the advent of code website. Did you want the name changing?

File: src/SonarPartTwo.php (hunk 2 above)
Comment (Gwen-Bradbury) [reply src/SonarPartTwo.php]: I've removed commented code

File: tests/SonarPartTwoTest.php (hunk 1 above)
Comment (Gwen-Bradbury) [reply tests/SonarPartTwoTest.php]: I've adjusted so the tests input the correct values.

File: src/SonarPartTwo.php (hunk 4 above)
Comment (OYates) [reply src/SonarPartTwo.php:9]: doesn't have anything to do with a submarine

File: tests/SonarPartTwoTest.php [hunk 6]
```diff
@@ -7,8 +7,18 @@
 
 class SonarPartTwoTest extends TestCase
 {
-    public function  testGivenOneNumberReturnsZero(): void
+    public function  testGivenOneWindowReturnsSix(): void
     {
         $this->assertSame(6, (new SonarPartTwo())->SweepPartTwo("1\n2\n3"));
```
Comment (OYates) [inline tests/SonarPartTwoTest.php:12]: you aren't looking for the total you are looking to see if it increases or not, so in this case the answer would be 0

File: src/SonarPartTwo.php (hunk 4 above)
Comment (Gwen-Bradbury) [reply src/SonarPartTwo.php:9]: Agreed. Name changed to subData.

File: tests/SonarPartTwoTest.php (hunk 6 above)
Comment (Gwen-Bradbury) [reply tests/SonarPartTwoTest.php:12]: Re-written tests.

**Pull request merged**